### PR TITLE
IA-1522: manual cypress tests

### PR DESCRIPTION
**Pull request by olethanh**
See the comment from qgerome on the original PR https://github.com/BLSQ/iaso/pull/1048

### CODE REVIEW

**Comment by beygorghor** [comment]
@codeKameleon take a look here https://github.com/BLSQ/iaso-private/pull/1048 to have @qgerome feed-back on this

**Comment by codeKameleon** [comment]
@cypress

**Comment by codeKameleon** [comment]
@cypress

**Comment by codeKameleon** [comment]
@cypress

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -150,106 +150,4 @@ jobs:
...
-          SLACK_FOOTER: ${{  format('<https://github.com/{0}/commit/{1}/checks|🔗{2} logs> | <https://github.com/{0}/commit/{1}/checks|🔗 Commit {1}>', github.repository, github.sha, github.job) }}
-          MSG_MINIMAL: true
-        if: failure()
-      # after the test run completes
-      # store videos and any screenshots
-      # NOTE: screenshots will be generated only if E2E test failed
-      # thus we store screenshots only on failures
-      # Alternative: create and commit an empty cypress/screenshots folder
-      # to always have something to upload
-      - uses: actions/upload-artifact@v2
```
Comment (qgerome) [inline .github/workflows/main.yml:245]: Ca

**Pull request closed without merging**